src/ruby/puppetserver-lib/puppet/server/ast_compiler.rb

```diff
@@ -24,26 +24,79 @@ def self.compile(compile_options)
       end
 
       def self.compile_ast(code, compile_options)
+        # TODO: move requires to optimal place and guard against FOSS not having bolt lib code
+        # CODEREVIEW: where should these requires be, and what should happen if they are not found?
+        require 'bolt'
+        require 'bolt/target'
+        require 'bolt/apply_inventory'
+        require 'bolt/apply_target'
+        require 'bolt/apply_result'
+        require 'addressable'
```

Comment:
Again, this wont be required if https://github.com/puppetlabs/bolt/pull/1770 is accepted